Dockerfile.build

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:16.04
```

Comment:
Do you think this file (or the one below it) need to be present if it's getting built in CI?

I would at least want `build-env.sh` to be put somewhere other than the root of the directory if we're actually adding it to master.

Reply:
yeah, I was going to ask your opinion on that - I think we can just remove them?  unless we want to provide an alternate way of generating build packages manually...